packages/compass/src/app/components/home.tsx

```diff
@@ -125,8 +83,7 @@ function Home({
   hideCollectionSubMenu,
   showSettings,
 }: Omit<HomeProps, 'connectionStorage'>): React.ReactElement | null {
-  const appRegistry = useLocalAppRegistry();
-  const { connectionInfo, isConnected, disconnect } =
+  const { isConnected, disconnect } =
```

Comment:
If we're making it impossible to use single connection mode by removing the form, why are we keeping this code here?

Reply:
I have reverted this bit and will be only removing the tests here. as much as I'd love to follow this rabbit hole, I don't have time to do that, so I'll leave it for the scheduled MC cleanup. I hope the partial cleanup I've done in the connection-form is okay. This is why I was asking in the channel if it's safe to say we won't be reverting the MC flag.